answer.js

```diff
@@ -0,0 +1,80 @@
+// 1.answer
+
+var p1 = 6;
+var p2 = 7;
+var p3 = 10;
+ function tripleFunction (p1,p2,p3) {
+   return p1 + p2 + p3
+ }
+
+tripleFunction(p1,p2,p3);
+
+// 2.answer
+
+function colorCar(color){
+  console.log( 'a' +''+ color +''+ 'car');
+}
+
+colorCar("red");
+
+//3.answer
+
+const aboutMe ={
+  name : 'Elif',
+  surname : 'Yasar',
+  age: '19',
+  nationality: 'Turkish',
+  address: {
+    street: 'Hello Street',
+    city: 'Toronto'
+  },
+}
+
+function learnAboutMe(){
+  console.log(aboutMe);
+}
+
+learnAboutMe();
+
+// 4.answer
+
+function vehcileType(color,code){
+  if (code === 1){
+    console.log('a'+ color + 'car')
+  }
+  else {
+    console.log('a'+ color + 'motorbike')
+  }
+}
+ vehcileType('blue', 1);
+
+ //5.answer
+
+ console.log(3 === 3 ? 'yes' : 'no');
+
+ //6.answer
+
+ let code = ["car" , "motorbike"]
+ function vehicle(color,age,code){
```

Comment:
I think you combined questions 6 and 9 but that's okay!
Question 9 involves writing a function with three parameters, `color`, `age`, and `code`. `code` should be a number (array index) that you use to get the value from an array. It shouldn't be the actual element from the array.
For example, you are calling your function like this:
```javascript
vehicle("blue", 3 , code[0]); // the third parameter, code[0], is the string "car" from your array on line 57
```
The function should be called like this:
```javascript
vehicle("blue", 3, 0); // the third parameter is a number (array index)
```
Then in the function, you should use the array index to get the vehicle type from the array.